lib/matplotlib/tests/test_figure.py

```diff
@@ -205,6 +208,7 @@ def test_axes_remove():
     assert_equal(len(fig.axes), 3)
 
 
+@cleanup
```

Comment:
Contrary to its documentation, `plt.figaspect` doesn't actually create a new figure, so this shouldn't affect anything.